packages/smart-wallet/test/test-walletFactory.js

```diff
@@ -72,8 +71,11 @@ test('bridge handler', async t => {
     ),
     blockTime: 0,
     blockHeight: 0,
+  };
+  assert(t.context.sendToBridge);
+  await t.throwsAsync(t.context.sendToBridge(validMsg), {
```

Comment:
again, I'd like to be sure the bridge protocol doesn't do something drastic like crashing the kernel when a bridge call throws / rejects.

Reply:
ok... found it. Nothing drastic. Just an unhandled rejection.

https://github.com/Agoric/agoric-sdk/blob/748d52cf888981c8c48c26a807fe5abf12ed4130/packages/vats/src/bridge.js#L40